packages/next/src/withTRPC.tsx

```diff
@@ -90,7 +92,9 @@ export function withTRPC<
     ssrContext: TSSRContext;
   };
   return (AppOrPage: NextComponentType<any, any, any>): NextComponentType => {
-    const trpc = createReactQueryHooks<TRouter, TSSRContext>();
+    const trpc = createReactQueryHooks<TRouter, TSSRContext>({
+      unstable_overrides: opts.config({}).unstable_overrides,
```

Comment:
this is a bit hacky, not sure what the best approach is to pass the options down?

Reply:
maybe they aren't even needed here? is the hooks created in `createTRPCNext`(see above) the important ones?

Reply:
This is fine 🤷